src/providers/twitch/twitchapi.cpp

```diff
@@ -0,0 +1,37 @@
+#include "providers/twitch/twitchapi.hpp"
+
+#include "Application.hpp"
+#include "common/UrlFetch.hpp"
+#include "controllers/accounts/AccountController.hpp"
+#include "providers/twitch/TwitchCommon.hpp"
+
+#include <QString>
+#include <QThread>
+
+namespace chatterino {
+
+void TwitchApi::FindUserId(const QString user, std::function<void(QString)> callback)
```

Comment:
Follow the normal function and method naming convention, which would in this case be `findUserId`